assests/js/index.js

```diff
@@ -0,0 +1,217 @@
+//Variable declaration
+const dateText = $("#currentDay");
+const timeText = $("#currentTime");
+const timeBlock = $(".container");
+const currentHr = moment().hour();
+
+const scheduleKey = "schedules";
+
+// use moment to get current date & time in format & real time update
+const getCurrentDate = function () {
+  const timer = function () {
+    const date_Time = moment();
```

Comment:
Try to keep variable names camel case in JS e.g. dateTime

Reply:
fixed the variable name 
thank you